Makefile

```diff
@@ -1,7 +1,7 @@
 GOTOOLS = \
 	github.com/mitchellh/gox \
 	github.com/golang/dep/cmd/dep \
-	gopkg.in/alecthomas/gometalinter.v2 \
+	github.com/alecthomas/gometalinter \
```

Comment:
this will pull latest master, which can cause non-determinism in our linting

Reply:
Metalinter is still hash locked in the script file. You only get different versions with update tools. (Though I could make update tools just call get tools as well)